editoast/src/views/rolling_stock/towed.rs

```diff
@@ -0,0 +1,610 @@
+
+        let towed_rolling_stocks: TowedRollingStockCountList = app
+            .fetch(app.get("/towed_rolling_stock"))
+            .assert_status(StatusCode::OK)
+            .json_into();
+
+        assert!(!towed_rolling_stocks.results.is_empty());
+    }
+
+    #[rstest]
+    async fn get_unknown_towed_rolling_stock() {
+        let app = TestAppBuilder::default_app();
+
+        let id: i64 = rand::random();
+
+        app.fetch(app.get(&format!("/towed_rolling_stock/{id}")))
+            .assert_status(StatusCode::NOT_FOUND);
+    }
+
+    #[rstest]
+    async fn create_and_get_towed_rolling_stock() {
+        let app = TestAppBuilder::default_app();
+
+        let name = Uuid::new_v4().to_string();
+        let towed_rolling_stock_json = json!({
+            "name": name,
+            "locked": false,
+            "mass": 42000,
+            "length": 16500,
+            "comfort_acceleration": 0.05,
+            "startup_acceleration": 0.05,
+            "inertia_coefficient": 1.0,
+            "rolling_resistance": {
+                "type": "yoda",
+                "A": 1000.0,
+                "B": 100.0,
+                "C": 10.0,
+            },
+            "gamma": {
+                "type": "CONST",
+                "value": 1.0,
+            },
+        });
+
+        let request = app
+            .post("/towed_rolling_stock")
+            .json(&towed_rolling_stock_json);
+
+        let created_towed_rolling_stock: TowedRollingStock =
+            app.fetch(request).assert_status(StatusCode::OK).json_into();
+        assert_eq!(created_towed_rolling_stock.name, name);
+
+        let id = created_towed_rolling_stock.id;
+
+        let get_towed_rolling_stock: TowedRollingStock = app
+            .fetch(app.get(&format!("/towed_rolling_stock/{id}")))
+            .assert_status(StatusCode::OK)
+            .json_into();
+
+        assert_eq!(get_towed_rolling_stock.name, name);
+    }
+
+    #[rstest]
+    async fn modify_towed_rolling_stock() {
+        let app = TestAppBuilder::default_app();
+
+        let name = Uuid::new_v4().to_string();
+        let mut towed_rolling_stock_json = json!({
+            "name": name,
+            "locked": false,
+            "mass": 42000,
+            "length": 16500,
+            "comfort_acceleration": 0.05,
+            "startup_acceleration": 0.05,
+            "inertia_coefficient": 1.0,
+            "rolling_resistance": {
+                "type": "yoda",
+                "A": 1000.0,
+                "B": 100.0,
+                "C": 10.0,
+            },
+            "gamma": {
+                "type": "CONST",
+                "value": 1.0,
+            },
+        });
+
+        let request = app
+            .post("/towed_rolling_stock")
+            .json(&towed_rolling_stock_json);
+
+        let towed_rolling_stock: TowedRollingStock =
+            app.fetch(request).assert_status(StatusCode::OK).json_into();
+        assert_eq!(towed_rolling_stock.name, name);
```

Comment:
Maybe we could create a towed rolling stock fixture or some `create_towed_rolling_stock()` method in the test module to avoid copy pasting the rolling stock create endpoint call in other tests that are not about this specific endpoint ?

Reply:
Done in be1d9c704c644fde51c1a716157514262bd8471d.